TaskHero/BottomBarLoader.swift

```diff
@@ -28,6 +29,9 @@ class BottomBarLoader: NSObject {
         bottomBarViewController.leftItemViewController = leftViewController
         bottomBarViewController.rightItemViewController = rightViewController
         
+        // save install and current user at loadtime
+        ParseClient.sharedInstance.connectCurrentUserAndInstallation()
```

Comment:
I don't think (or not sure how) the app delegate would work - we need this call after the user's logged in - and this loader is the first vc called.

Another logical spot is to call this on the login screen with successful credentials which actually might be better - I'll make this change.